src/plugins/discover/public/embeddable/initialize_search_embeddable_api.tsx

```diff
@@ -97,6 +97,7 @@ export const initializeSearchEmbeddableApi = async (
   const density$ = new BehaviorSubject<DataGridDensity | undefined>(initialState.density);
   const sort$ = new BehaviorSubject<SortOrder[] | undefined>(initialState.sort);
   const savedSearchViewMode$ = new BehaviorSubject<VIEW_MODE | undefined>(initialState.viewMode);
+  const breakdownField$ = new BehaviorSubject<string | undefined>(initialState.breakdownField);
 
   /**
```

Comment:
hmm , I'm not familar with the embeddable code here, but if we want to keep those fields, do we need to introduce observables for all of those, even if they don't change?  @jughosta , do you know ? thx

Reply:
I asked @Heenawter about this, and apparently this is the way to go.

Reply:
yes, that's a good question, because if this is the way to go, we would need to change this part of the code every time new properties are introduced. Which doesn not happen too often, but still ... I'd prefer a way that would take care of this automatically, @Heenawter is there an alternative to his ? thx!

Reply:
@kertal I guess I'm a bit confused what the goal is here. When you unlink a saved search, if that saved object has state that doesn't need to be used for the embeddable, why would we want to keep track of it **in** the embeddable? If you want to track a piece of state, the comparators for an embeddable require an observable because it's assuming that most state is "editable" - if it's state that doesn't change and isn't required for the embeddable to function, do you really need to track it? Why keep state around that you don't need? 🤷 

cc @nreese @ThomThomson

Reply:
I'm not exactly sure why you'd want to track all of the saved object state when the panel is no longer linked. In https://github.com/elastic/kibana/pull/203662 I also added a `rawSavedObjectAttributes` which tracks the attributes taken directly from the saved object. So if you _really_ need to retain that information I'd recommend storing it there.

On the other hand, if there isn't a user-facing reason for this change is it really worth doing? I'd let the data discovery team weigh in on that.

Reply:
let's pause until @davismcphee is back from PTO in the new year, who originally requested this change